wallet/res/layout/fragment_payments_pay.xml

```diff
@@ -16,6 +16,13 @@
         app:pb_sub_title="@string/payments_pay_to_contact_sub_title"
         app:pb_title="@string/payments_pay_to_contact_title" />
 
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/frequent_contacts_rv"
+        android:layout_marginStart="30dp"
```

Comment:
I'm wondering why this margin is 30dp wide, it adds awkward looking indentation:
![image](https://user-images.githubusercontent.com/8437401/90237614-d90e3f80-de24-11ea-8ce7-1c8353bb87af.png)

Shouldn't it be 15dp as in `@style/MenuRow`?

Reply:
That was a good question.  I am trying to copy this design, but there are many other differences that make it look differnet (no blue buttons, no rounded white rectangles for each MenuRow, etc.
![image (1)](https://user-images.githubusercontent.com/5956129/90256114-13161c00-ddfa-11ea-9c82-e54a56e6ed7d.png)

Reply:
Let's merge it as it is and fix during the UI polishing stage if needed